app/src/utils.js

```diff
@@ -1,5 +1,160 @@
+      return "Autre";
+    case "GENERAL_SCHOOL":
+      return "En enseignement général ou technologique";
+    case "PROFESSIONAL_SCHOOL":
+      return "En enseignement professionnel";
+    case "AGRICULTURAL_SCHOOL":
+      return "En lycée agricole";
+    case "SPECIALIZED_SCHOOL":
+      return "En établissement spécialisé";
+    case "APPRENTICESHIP":
+      return "En apprentissage";
+    case "EMPLOYEE":
+      return "Salarié(e)";
+    case "INDEPENDANT":
+      return "Indépendant(e)";
+    case "SELF_EMPLOYED":
+      return "Auto-entrepreneur";
+    case "ADAPTED_COMPANY":
+      return "En ESAT, CAT ou en entreprise adaptée";
+    case "POLE_EMPLOI":
+      return "Inscrit(e) à Pôle emploi";
+    case "MISSION_LOCALE":
+      return "Inscrit(e) à la Mission locale";
+    case "CAP_EMPLOI":
+      return "Inscrit(e) à Cap emploi";
+    case "NOTHING":
+      return "Inscrit(e) nulle part";
+    case "admin":
+      return "modérateur";
+    case "referent_department":
+      return "Référent départemental";
+    case "referent_region":
+      return "Référent régional";
+    case "COHESION_STAY":
+      return "Séjour de cohésion";
+    case "INSCRIPTION":
+      return "Inscription";
+    case "SUMMER":
+      return "Vacances d'été (juillet ou août)";
+    case "AUTUMN":
+      return "Vacances d'automne";
+    case "DECEMBER":
+      return "Vacances de fin d'année (décembre)";
+    case "WINTER":
+      return "Vacances d'hiver";
+    case "SPRING":
+      return "Vacances de printemps";
+    case "EVENING":
+      return "En soirée";
+    case "END_DAY":
+      return "En fin de journée";
+    case "WEEKEND":
+      return "Durant le week-end";
+    case "CITIZENSHIP":
+      return "Citoyenneté";
+    case "CULTURE":
+      return "Culture";
+    case "DEFENSE":
+      return "Défense et mémoire";
+    case "EDUCATION":
+      return "Éducation";
+    case "ENVIRONMENT":
+      return "Environnement";
+    case "HEALTH":
+      return "Santé";
+    case "SECURITY":
+      return "Sécurité";
+    case "SOLIDARITY":
+      return "Solidarité";
+    case "SPORT":
+      return "Sport";
+    case "UNIFORM":
+      return "Corps en uniforme";
+    case "OTHER":
+      return "Autre";
+    case "UNKNOWN":
+      return "Non connu pour le moment";
+    case "FIREFIGHTER":
+      return "Pompiers";
+    case "POLICE":
+      return "Police";
+    case "ARMY":
+      return "Militaire";
+    case "DURING_HOLIDAYS":
+      return "Sur les vacances scolaires";
+    case "DURING_SCHOOL":
+      return "Sur le temps scolaire";
+    case "true":
+      return "Oui";
+    case "false":
+      return "Non";
+    case "male":
+      return "Homme";
+    case "female":
+      return "Femme";
+    case "father":
+      return "Père";
+    case "mother":
+      return "Mère";
+    case "representant":
+      return "Représentant légal";
+    case "SERVER_ERROR":
+      return "Erreur serveur";
+    case "NOT_FOUND":
+      return "Existe pas";
```

Comment:
ça fait bizarre peut être : 

```suggestion
      return "Ressource introuvable";
```